WebApp2/src/main/resources/init.sql

```diff
@@ -0,0 +1,43 @@
+-- DDL for schema students
+
+DROP SCHEMA IF EXISTS students;
+CREATE SCHEMA IF NOT EXISTS students;
+
+DROP TABLE IF EXISTS students.students_db, students.students_city_db;
+CREATE TABLE IF NOT EXISTS students.students_db
+(
+    id         INT         NOT NULL AUTO_INCREMENT UNIQUE,
+    first_name varchar(50) NOT NULL,
+    last_name  varchar(50) NOT NULL,
+    course     INT         NOT NULL,
+    PRIMARY KEY (id)
+);
+CREATE TABLE IF NOT EXISTS students.students_city_db
+(
+    id          INT         NOT NULL AUTO_INCREMENT,
+    students_id INT         NOT NULL,
+    city_from   varchar(50) NOT NULL,
+    city_study  varchar(50) NOT NULL,
+    PRIMARY KEY (id),
+    FOREIGN KEY (students_id) REFERENCES students_db (id) ON DELETE CASCADE,
+    FOREIGN KEY (students_id) REFERENCES students_db (id) ON UPDATE CASCADE,
+    FOREIGN KEY (students_id) REFERENCES students_db (id)
+);
```

Comment:
CREATE TABLE IF NOT EXISTS students.students_city_db
(
    id          INT         NOT NULL AUTO_INCREMENT,
    students_id INT         NOT NULL,
    city_from   varchar(50) NOT NULL,
    city_study  varchar(50) NOT NULL,
    PRIMARY KEY (id),
    FOREIGN KEY (students_id) REFERENCES students_db (id) ON UPDATE CASCADE ON DELETE CASCADE
);